snap/db.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 CoreOS, Inc.
+// Copyright 2016 CoreOS, Inc.
```

Comment:
Sorry for the back & forth! I thought you meant only for the ioutil package. Will fix these.